spec/common/version_handler_spec.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+describe ReleaseManager::Common::VersionHandler do
+  subject { build(:version_handler) }
+
+    it { is_expected.to respond_to(:should_add_versions?) }
+    it { is_expected.to respond_to(:change_version) }
+    it { is_expected.to respond_to(:add_versions) }
+
+  context 'version management with x release type' do
+    it 'changes version' do
+      expect(subject.change_version('1.12.9')).to eq('1.12.10')
+    end
+  end
+
+  context 'version management with y release type' do
+    subject { build(:version_handler, release_type: 'y') }
+
+    it 'changes version' do
+      expect(subject.change_version('1.9.1')).to eq('1.10.0')
+    end
+  end
+
+  context 'component picking' do
+    it 'should ignore component' do
```

Comment:
Avoid should syntax, the convention is to do something like `it 'ignores component'`